tests/unit/test_cli.py

```diff
@@ -372,6 +373,52 @@ def test_specify_packages(
         with mock_fetch_deps(expect_request):
             invoke_expecting_sucess(app, ["fetch-deps", package_arg])
 
+    @pytest.mark.parametrize(
+        "package_arg, expect_packages",
+        [
+            (
+                {"type": "gomod"},
+                [{"type": "gomod"}],
+            ),
+            (
+                {"packages": [{"type": "gomod"}]},
+                [{"type": "gomod"}],
+            ),
+            (
+                {
+                    "packages": [
+                        {"type": "gomod", "path": "pkg_a"},
+                        {"type": "gomod", "path": "pkg_b"},
+                    ]
+                },
+                [
+                    {"type": "gomod", "path": "pkg_a"},
+                    {"type": "gomod", "path": "pkg_b"},
+                ],
+            ),
```

Comment:
when adding parametrize, please always try to pair a test case with some test ID, it makes the test output more readable and a bit easier to filter.